node/pkg/processor/processor.go

```diff
@@ -139,6 +140,14 @@ type Processor struct {
 	acctReadC      <-chan *common.MessagePublication
 	pythnetVaas    map[string]PythNetVaaEntry
 	gatewayRelayer *gwrelayer.GatewayRelayer
+	updateVAALock  sync.Mutex
+	updatedVAAs    map[string]*updateVaaEntry
```

Comment:
Would it be useful to anyone to have a prometheus metric showing how big this map gets?

Reply:
I kinda don't think so. I think there are a lot of metrics that no one ever looks at, so I don't really want to add to that. . .